README.md

```diff
@@ -1,2 +1,56 @@
+
+## Plastic
+- Use cloth bags instead of disposable plastic ones. [Example](https://www.ecobags.com/Our_Products/Produce_Bags)
+
+- Invest in one quality steel bottle and refill ot with water instead of using plastic bottles. Ask in the cafe bar to refill it (if they reject it they are not cool) or use a fountain in the city you are currently in
+
+- Use wooden tooth brush, plastic-less ear sticks, solid soap (doesn't come in plastic bottle), powder detergent packed in carton instead of liquid one (or even better eco-egg)
+
+- Know your plastic, not all the plastic is recycleable. Even during recycling part of it ends up being micro-plastic and the recycling procedure itself has a carbon footprint
+
+## General shopping
+- When you decide to buy something non-essential, postpone it for 2 weeks and then re-think about if you still need it
+
+- Do the research about the manufacturer whose product you are buying, what materials is it using, how is it treating its employees, where is it doing the manufacturing
+
+- Try to see through marketing, don't fall for it, don't shop to feel happier, block ads, chose the content you consume 
+
+- Prefer zero-waste manufacturers, but don't trust them just because they say they are zero-waste, investigate it
+
+- Buy quality products that will last long, don't have plastic and are more easily repairable, modular or upgradeable. Most of the products today are designed for waste, they are planned to become obsolete
+
+- Be mindful about greenwashing - big companies offering same products under different green brand, or telling they are giving away an X amount of money to something for every buck you spend
+
+- Separate and recycle your waste (but try to make less waste)
+
+## Information technology
+- Psihology, bubbles, don't compare yourself to others, because you are comparing yourself to the picture others present about themselves on social networks which is false. Social networks don't promote sustainable psychology
+
+- Similar to phisical products designed for waste, most of the news portals and individual content generators are under preassure to deliver more content regardless of quality, so ignore them, they are wasting your valuable time. Investigate and choose just a couple of content producers that deliver less, but maintain quality. Think about supporting them on Patreon or buying subscription if it's a magazine or portal
+
+- See if there are content producers that suit you and that try to use as less energy as possible, [this one for example](https://www.lowtechmagazine.com/)
+
+## TBD
+- There needs to be some fact-checking done to see how much can individuals influence sustainibility versus big companies. In other words, does it make sense for me to fight when big companies pollute "in a day more than I can save in a year". Even if it's true it makes sense to put political preassure to punish such a companies and economical preassure directly on those companies by not buying their products
```

Comment:
I wouldn't question this since. Just do the best you can on daily basis. The effort for everything stated above is low and benefit is great. You may even save money in the process.